content/docs/guides/autoscaling-guide.md

```diff
@@ -0,0 +1,57 @@
+1. Under **Compute size**, select the **Autoscaling** option.
+1. Using the slider, specify a minimum and maximum compute size.
+    ![Autoscaling edit settings](/docs/guides/autoscaling_edit_settings.png)
+
+    Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. _Autoscaling_ currently supports a range of 1/4 (.25) to 7 vCPUs. One vCPU has 4 GB of RAM, 2 vCPUs have 8 GB of RAM, and so on. The amount of RAM in GBs is always 4 times the number of vCPUs.
+
+    <Admonition type="note">
+    You can also configure the **Auto-suspend delay** setting for your compute endpoint at the same time. The **Auto-suspend delay** setting defines the number of seconds of inactivity after which a compute endpoint is automatically suspended. This feature is also referred to as "scale-to-zero".
+    </Admonition>
+1. Click **Save**.
+
+## Set Autoscaling defaults when creating a project
+
+A project is a top-level object in the Neon object hierarchy. You can think of it as a container for all other objects, including branches and compute endpoints.
+
+Enabling Autoscaling when you create a project allows you to set _Autoscaling_ default settings for all compute endpoints created in your project. You can adjust _Autoscaling_ settings for individual compute endpoints afterward, but setting a desired defaults when creating a project will save you from having to configure these settings for each compute later on.
+
+To configure Autoscaling default settings at project creation time:
+
+1. Navigate to the [Neon Console](https://console.neon.tech).
+2. If you are creating your very first project, click **Create a project**. Otherwise, click **New Project**.
+3. Specify a name, a PostgreSQL version, and a region.
+4. Under **Compute size**, select the **Autoscaling** option.
+5. Using the slider, specify a minimum and maximum compute size.
+    ![Autoscaling](/docs/guides/autoscaling_project_creation.png)
+
+    <Admonition type="note">
+    You can also specify a default **Auto-suspend delay** setting at project creation time. The **Auto-suspend delay** setting defines the number of seconds of inactivity after which a compute endpoint is automatically suspended. This feature is also referred to as "scale-to-zero". The setting specified at project creation time becomes the default setting for newly created compute endpoints.
+    </Admonition>
+
+6. Click **Create Project**. Your initial compute endpoint is created with the specified settings. All compute endpoints created in the future when creating a branch or adding a compute endpoint to a branch are created with the settings you specified.
+
+## Monitor Autoscaling
+
+The `neon_utils` extension provides a `num_cpus()` function for monitoring how the _Autoscaling_ feature allocates compute resources in response to workload. For more information, see [The neon_utils extension](/docs/extensions/neon-utils).
```

Comment:
do you warn about fractional CUs here as well?

Reply:
I think we're OK to leave that detail in the neon-utils page. User will have to go there to find out how to use it anyway.